addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -223,32 +215,34 @@ Wysiwyg.include({
             .dropdown({});
         return toggleDropdown($megaMenuToggles, false);
     },
+    /**
+     * Toggles the mega menu
+     *
+     * @private
+     * @returns {Promise}
+     */
+    _toggleMegaMenu: function (toggleEl) {
+        const megaMenuEl = toggleEl.parentElement.querySelector('.o_mega_menu');
+        if (!megaMenuEl || !megaMenuEl.classList.contains('show')) {
+            return this.snippetsMenu._activateSnippet(false);
+        }
+        return this.snippetsMenu._activateSnippet($(megaMenuEl));
```

Comment:
Make the method public instead of private to be able to do that properly. Or maybe create a new public method calling the private one to avoid the diff (and you could omit the last two parameters in the public method).